File: output/ChatGPT/avatar/Java/C++/atcoder_ABC171_B.cpp
#include <iostream>
#include <algorithm>
#include <vector>

using namespace std;

int main() {
    int n, k;
    cin >> n >> k;
    vector<int> a(n);
    for (int i = 0; i < n; ++i) {
        cin >> a[i];
    }
    sort(a.begin(), a.end());
    int ans = 0;
    for (int i = 0; i < k; ++i) {
        ans += a[i];
    }
    cout << ans << endl;

}

File: output/ChatGPT/avatar/Java/Python/atcoder_ABC171_B.py
import sys


def solve(n, k, a):
    a.sort()
    ans = 0
    for i in range(k):
        ans += a[i]
    return ans


def main():
    n, k = map(int, input().split())
    a = list(map(int, input().split()))
    ans = solve(n, k, a)
    print(ans)


if __name__ == "__main__":
    main()